projects/plugins/boost/app/features/image-guide/src/types.ts

```diff
@@ -0,0 +1,28 @@
+export type MeasuredImage = {
```

Comment:
Would it make sense to define this as `MeasuredImage = Image & {...` since MeasuredImage seems to contain all the same elements, then some?

Reply:
It would, I kept those files separate because wasn't sure how they'd be reused. I think I'll settle this down a little bit after this gets merged and we know what we're doing server-side.

Reply:
I'll keep these separate, but this is something that should be refactored later.